arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -430,47 +428,33 @@ def checkFootnote(elt: ModelInlineFootnote | ModelResource, text: str) -> None:
-                                        if not elt.modelXbrl.fileSource.isInArchive(normalizedUri):
-                                            normalizedUri = elt.modelXbrl.modelManager.cntlr.webCache.getfilename(normalizedUri)
-                                        imglen = 0
-                                        with elt.modelXbrl.fileSource.file(normalizedUri,binary=True)[0] as fh:
-                                            imgContents = fh.read()
-                                            imglen += len(imgContents)
-                                            checkImageContents(modelXbrl, elt, os.path.splitext(src)[1], True, imgContents)
-                                            imgContents = None # deref, may be very large
-                                        #if imglen < browserMaxBase64ImageLength:
-                                        #    modelXbrl.error("ESEF.2.5.1.imageIncludedAndNotEmbeddedAsBase64EncodedString",
-                                        #        _("Images MUST be included in the XHTML document as a base64 encoded string unless their size exceeds support of browsers (%(maxImageSize)s): %(file)s."),
-                                        #        modelObject=elt, maxImageSize=browserMaxBase64ImageLength, file=os.path.basename(normalizedUri))
-                                    except IOError as err:
-                                        modelXbrl.error("ESEF.2.5.1.imageFileCannotBeLoaded",
-                                            _("Image file which isn't openable '%(src)s', error: %(error)s"),
-                                            modelObject=elt, src=src, error=err)
+                                # presume it to be an image file, check image contents
+                                try:
+                                    base = elt.modelDocument.baseForElement(elt)
+                                    normalizedUri = elt.modelXbrl.modelManager.cntlr.webCache.normalizeUrl(src, base)
+                                    if not elt.modelXbrl.fileSource.isInArchive(normalizedUri):
+                                        normalizedUri = elt.modelXbrl.modelManager.cntlr.webCache.getfilename(normalizedUri)
+                                    imglen = 0
+                                    with elt.modelXbrl.fileSource.file(normalizedUri,binary=True)[0] as fh:
+                                        imgContents = fh.read()
+                                        imglen += len(imgContents)
+                                        checkImageContents(modelXbrl, elt, os.path.splitext(src)[1], True, imgContents)
+                                        imgContents = None # deref, may be very large
+                                    #if imglen < browserMaxBase64ImageLength:
+                                    #    modelXbrl.error("ESEF.2.5.1.imageIncludedAndNotEmbeddedAsBase64EncodedString",
```

Comment:
Would it be better to have a toggle that is off by default if we are going to keep the comment around? The commented out code seems not that useful as its already in git history